cluster/gce/configure-vm.sh

```diff
@@ -77,12 +77,13 @@ function set-kube-env() {
   done
 
   # kube-env has all the environment variables we care about, in a flat yaml format
-  eval $(python -c '''
+  eval "$(python -c '
 import pipes,sys,yaml
 
 for k,v in yaml.load(sys.stdin).iteritems():
-  print "readonly {var}={value}".format(var = k, value = pipes.quote(str(v)))
-''' < "${kube_env_yaml}")
+  print """readonly {var}={value}""".format(var = k, value = pipes.quote(str(v)))
+  print """export {var}""".format(var = k)
```

Comment:
This is required. I need to export these vars because I need them in `kubernetes/saltbase/install.sh`. Before the quoting changes I don't think this was working as expected either